server/sonar-web/src/main/js/libs/application.js

```diff
@@ -99,7 +99,7 @@
 
 function toggleFav (resourceId, elt) {
   jQuery.ajax({
-    type: 'POST', dataType: 'json', url: '/favourites/toggle/' + resourceId,
+    type: 'POST', dataType: 'json', url: window.baseUrl + '/favourites/toggle/' + resourceId,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3512)